MobileCenter/MobileCenter/Internals/Sender/MSIngestionSender.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ */
```

Comment:
we have the header more or less everywhere but not in tests. If we don't need it, let's just get rid of it in a sep. PR.